ginga/canvas/types/basic.py

```diff
@@ -89,8 +89,6 @@ def draw(self, viewer):
         cx, cy = viewer.get_canvas_xy(x, y)
         cr.draw_text(cx, cy, self.text, rot_deg=self.rot_deg)
 
-        if self.showcap:
-            self.draw_caps(cr, self.cap, cpoints)
```

Comment:
Note: I can't figure out where `cpoints` is supposed to come from, so I just removed this line entirely, assuming that this is some copy-and-paste artifact. Is this the right approach?